force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +140,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
+        Integer numberOfVowels = 0;
+        List<string> lettersInWord = new List<String>();
 
-        /*
-        for () {
+        for (Integer i = 0; i < str.length(); i++) {
             // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
+            lettersInWord.add(str.substring(i, i+1));            
+        }
+        
+        for (String letter : lettersInWord) {
```

Comment:
You should be able to simplify this to a single for loop by setting letter= str.substring(i, i+1) in the first loop and then moving the if statement inside